hedera-node/hedera-app/src/test/java/com/hedera/node/app/grpc/impl/usage/GrpcUsageTrackerTest.java

```diff
@@ -0,0 +1,226 @@
+        assertEquals(2, counter.sum());
+
+        assertEquals(usageBucket.time(), Instant.parse("2025-04-03T15:30:00.000Z"));
+    }
+
+    @Test
+    void testLogOutput() {
+        final Clock clock = Clock.fixed(Instant.parse("2025-04-03T15:32:32.426457Z"), ZoneOffset.UTC);
+        final GrpcUsageTrackerConfig config = new GrpcUsageTrackerConfig(true, 15, 100);
+        final ConfigProvider configProvider = mock(ConfigProvider.class);
+        final VersionedConfiguration configuration = mock(VersionedConfiguration.class);
+        final ServerCall<String, String> serverCall = mock(ServerCall.class);
+        final ServerCallHandler<String, String> handler = mock(ServerCallHandler.class);
+        final Metadata javaMetadata = new Metadata();
+        javaMetadata.put(userAgentHeaderKey, "hiero-sdk-java/2.3.1");
+        final Metadata goMetadata = new Metadata();
+        goMetadata.put(userAgentHeaderKey, "hiero-sdk-go/1.5.6");
+        final Metadata luaMetadata = new Metadata();
+        luaMetadata.put(userAgentHeaderKey, "hiero-sdk-lua/0.0.1");
+
+        final MethodDescriptor<String, String> commitDescriptor = newDescriptor("proto.MyService/commit");
+        final MethodDescriptor<String, String> getDescriptor = newDescriptor("proto.MyService/get");
+
+        when(configProvider.getConfiguration()).thenReturn(configuration);
+        when(configuration.getConfigData(GrpcUsageTrackerConfig.class)).thenReturn(config);
+        when(serverCall.getMethodDescriptor())
+                .thenReturn(commitDescriptor, getDescriptor, commitDescriptor, commitDescriptor);
+
+        final GrpcUsageTracker usageTracker = new GrpcUsageTracker(configProvider, clock);
+
+        // make two calls
+        assertDoesNotThrow(() -> usageTracker.interceptCall(serverCall, javaMetadata, handler)); // MyService:Commit
+        assertDoesNotThrow(() -> usageTracker.interceptCall(serverCall, goMetadata, handler)); // MyService:Get
+        assertDoesNotThrow(() -> usageTracker.interceptCall(serverCall, luaMetadata, handler)); // MyService:Commit
+        assertDoesNotThrow(() -> usageTracker.interceptCall(serverCall, javaMetadata, handler)); // MyService:Commit
+
+        assertDoesNotThrow(usageTracker::logAndResetUsageData); // log the usage data out
+
+        List<String> logs = accessLogCaptor.infoLogs();
+        assertEquals(3, logs.size());
+        final String log1 =
+                "|time=2025-04-03T15:30:00Z|service=MyService|method=Commit|sdkType=HieroSdkJava|sdkVersion=2.3.1|count=2|";
+        final String log2 =
+                "|time=2025-04-03T15:30:00Z|service=MyService|method=Commit|sdkType=Unknown|sdkVersion=Unknown|count=1|";
+        final String log3 =
+                "|time=2025-04-03T15:30:00Z|service=MyService|method=Get|sdkType=HieroSdkGo|sdkVersion=1.5.6|count=1|";
+
+        assertTrue(logs.contains(log1));
+        assertTrue(logs.contains(log2));
+        assertTrue(logs.contains(log3));
```

Comment:
Not sure what value there is in defining `log1` etc.

```suggestion
        assertTrue(logs.contains("|time=2025-04-03T15:30:00Z|service=MyService|method=Commit|sdkType=HieroSdkJava|sdkVersion=2.3.1|count=2|"));
        assertTrue(logs.contains("|time=2025-04-03T15:30:00Z|service=MyService|method=Commit|sdkType=Unknown|sdkVersion=Unknown|count=1|"));
        assertTrue(logs.contains("|time=2025-04-03T15:30:00Z|service=MyService|method=Get|sdkType=HieroSdkGo|sdkVersion=1.5.6|count=1|"));
```